pkg/visibility/server.go

```diff
@@ -60,6 +60,8 @@ var (
 // +kubebuilder:rbac:groups=flowcontrol.apiserver.k8s.io,resources=flowschemas/status,verbs=patch
 
 // CreateAndStartVisibilityServer creates visibility server injecting KueueManager and starts it
+//
+//nolint:revive // deep-exit: backwards compatibility
```

Comment:
Can we also eliminate this exclusion? If this is much work or is risky we can ticket separately